src/m365/aad/commands/user/user-hibp.ts

```diff
@@ -0,0 +1,100 @@
+import { Logger } from '../../../../cli';
+import {
+  CommandOption
+} from '../../../../Command';
+import Command from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import commands from '../../commands';
+import Utils from '../../../../Utils';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  userName: string;
+  apiKey: string;
+  domain?: string
+}
+
+class HIBPCommand extends Command {
```

Comment:
Command's class name should follow the naming convention of ServiceCommandCommand, eg. `AadUserHIBPCommand`